tests/validate/common.mk

```diff
@@ -0,0 +1,116 @@
+#lowlevel suite common makefile definitions
+
+ifdef XILINX_SDACCEL
+ XILINX_VITIS := $(XILINX_SDACCEL)
+endif
```

Comment:
Can we clean this up?  There is no SDACCEL, do we need to support folks setting this var?